plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -640,17 +640,25 @@ function ListMenuItem:update()
             if self.file_deleted then -- unless file was deleted (can happen with History)
                 hint = " " .. _("(deleted)")
             end
+            local text_widget
+            local fontsize_deleted = _fontSize(18)
```

Comment:
It's not (only) for items deleted.
It's fot item with no bookinfo (yet) in DB - and it's probably gonna be fetched in a few seconds.
So better name this `fontsize_no_bookinfo`, or just `fontsize` as it's just a local variable.

Reply:
It was the only way I could produce this situation, hence the wrong assumption.

Reply:
Well, you can produce it by just changing the "File per page" value :) Then as you browse, the cover_sizetag being different, bookinfo is dropped, and you're in a "no bookinfo" situation. That's the temporary display while the covers and metadata are fetched.
(If you don't see them, you must have a super fast device! :)